v0.1.10 (#11)
rapdev-io/datadog-secret-backend: docs/azure/README.md

| azure_certificate_path | Path to Application Azure Certificate |
| azure_certificate_password | Password for Azure Certificate |

In all cases, you will need to specify `keyvaulturl` and with service principal based authentication, the `azure_tenant_id` and `azure_client_id` corresponding to the Azure KeyVault resource.

> Simple string values can be defined adding the config variable `force_string: true`. The `force_string: true` backend configuration setting will interpret the contents of secret as a string regardless of input when defining AWS Secrets Manager or Azure Key Vault backends.

Author: ```suggestion
Simple string values can be defined adding the config variable `force_string: true`. The `force_string: true` backend configuration setting will interpret the contents of the Azure Key Vault Secret as a string, even if the stored secret value is valid JSON.
```

> This will allow for generating simple string values with or without JSON parsing using the _ secretID. All input will be interpreted as a single string and if it fails to unmarshal the JSON, it will grab the raw string from the input. The secretId then gets coerced as _ and will access DD with the backendID and its secret using the ENC[] notation.

Author: ```suggestion
When `force_string: false` is defined, or when the backend setting `force_string` is not defined, then the secret value will be interpreted as JSON with simple depth of one (1) and create a secretId with each field name. If the secret value is not valid JSON, then it will behave as if `force_string: true` was defined and will return the full contents of the secret value with secretId `_`.
```

| azure_certificate_path | Path to Application Azure Certificate |
| azure_certificate_password | Password for Azure Certificate |

> In all cases you'll need to specify `keyvaulturl`, and in any service principal based authentication, `azure_tenant_id` and `azure_client_id` to correspond to the Azure KeyVault reasource and the application definition being used to authenticate to Azure.

Author: ```suggestion
In all cases, you will need to specify `keyvaulturl` and with service principal based authentication, the `azure_tenant_id` and `azure_client_id` corresponding to the Azure KeyVault resource.
```

> Simple string values can be defined adding the config variable `force_string: true`. This will interpret the contents of secret as a string regardless of input.

Author: ```suggestion
Simple string values can be defined adding the config variable `force_string: true`. The `force_string: true` backend configuration setting will interpret the contents of secret as a string regardless of input when defining AWS Secrets Manager or Azure Key Vault backends.
```
